osu.Game/Configuration/TaikoTouchControlType.cs

```diff
@@ -0,0 +1,14 @@
+// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+// See the LICENCE file in the repository root for full licence text.
+
+#nullable disable
+
+namespace osu.Game.Configuration
+{
+    public enum TaikoTouchControlType
+    {
+        Default, // KDDK
```

Comment:
Would probably call this by it's actual name than "Default".

Reply:
Updated to this just before the PR was closed, I assumed that "Default" would be more user-friendly to someone who doesn't know the terminology but ah well